EventId,EventTemplate,Occurrences
073d92ad,iommu: DMA domain TLB invalidation policy: lazy mode,1
6f935fcb,"Current system time is further ahead 15y after build time, but cannot correct: Invalid argument",2
a4769158,"<*> Failed to add dependency on rpcbind.sockets, ignoring: Invalid argument",2
1c90e8a6,[drm:dc_link_detect_helper [amdgpu]] *ERROR* No EDID read.,1
0fbf7307,ERROR: Could not statfs: No such file or directory,1
bb6a31d4,Failed to open /etc/resolv.conf: Invalid argument,1
b59aac41,"(WW) warning, (EE) error, (NI) not implemented, (??) unknown.",2
11d8c3ee,"<warn> [1699535499.3895] keyfile: load: ""/etc/NetworkManager/system-connections/Wired connection 1.nmconnection"": failed to load connection: invalid connection: connection.interface-name: 'place_holder_nmcli': interface name is longer than 15 characters",1
0784ed1c,"time=""2023-11-09T13:11:39.642724531Z"" level=info msg=""skip loading plugin \""io.containerd.snapshotter.v1.aufs\""..."" error=""aufs is not supported (modprobe aufs failed: exit status 1 \""modprobe: FATAL: Module aufs not found in directory /lib/modules/5.15.73\\n\""): skip plugin"" type=io.containerd.snapshotter.v1",1
31aa37bb,"time=""2023-11-09T13:11:39.642828135Z"" level=warning msg=""failed to load plugin io.containerd.snapshotter.v1.devmapper"" error=""devmapper not configured""",1
ffcf5908,"time=""2023-11-09T13:11:39.643352561Z"" level=info msg=""skip loading plugin \""io.containerd.snapshotter.v1.zfs\""..."" error=""path /var/lib/containerd/io.containerd.snapshotter.v1.zfs must be a zfs filesystem to be used with the zfs snapshotter: skip plugin"" type=io.containerd.snapshotter.v1",1
7d18a44e,"time=""2023-11-09T13:11:39.643441228Z"" level=warning msg=""could not use snapshotter devmapper in metadata plugin"" error=""devmapper not configured""",1
78e841d9,"time=""2023-11-09T13:11:39.695874707Z"" level=info msg=""skip loading plugin \""io.containerd.tracing.processor.v1.otlp\""..."" error=""no OpenTelemetry endpoint: skip plugin"" type=io.containerd.tracing.processor.v1",1
ff16350b,"time=""2023-11-09T13:11:39.695994893Z"" level=error msg=""failed to initialize a tracing processor \""otlp\"""" error=""no OpenTelemetry endpoint: skip plugin""",1
fd9384f2,"time=""2023-11-09T13:11:39.697034708Z"" level=info msg=""Start cri plugin with config {PluginConfig:{ContainerdConfig:{Snapshotter:overlayfs DefaultRuntimeName:runc DefaultRuntime:{Type: Path: Engine: PodAnnotations:[] ContainerAnnotations:[] Root: Options:map[] PrivilegedWithoutHostDevices:false BaseRuntimeSpec: NetworkPluginConfDir: NetworkPluginMaxConfNum:0} UntrustedWorkloadRuntime:{Type: Path: Engine: PodAnnotations:[] ContainerAnnotations:[] Root: Options:map[] PrivilegedWithoutHostDevices:false BaseRuntimeSpec: NetworkPluginConfDir: NetworkPluginMaxConfNum:0} Runtimes:map[runc:{Type:io.containerd.runc.v2 Path: Engine: PodAnnotations:[] ContainerAnnotations:[] Root: Options:map[BinaryName: CriuImagePath: CriuPath: CriuWorkPath: IoGid:0 IoUid:0 NoNewKeyring:false NoPivotRoot:false Root: ShimCgroup: SystemdCgroup:false] PrivilegedWithoutHostDevices:false BaseRuntimeSpec: NetworkPluginConfDir: NetworkPluginMaxConfNum:0}] NoPivot:false DisableSnapshotAnnotations:true DiscardUnpackedLayers:false IgnoreRdtNotEnabledErrors:false} CniConfig:{NetworkPluginBinDir:/opt/cni/bin NetworkPluginConfDir:/etc/cni/net.d NetworkPluginMaxConfNum:1 NetworkPluginConfTemplate: IPPreference:} Registry:{ConfigPath: Mirrors:map[] Configs:map[] Auths:map[] Headers:map[]} ImageDecryption:{KeyModel:node} DisableTCPService:true StreamServerAddress:<*> StreamServerPort:0 StreamIdleTimeout:4h0m0s EnableSelinux:false SelinuxCategoryRange:1024 SandboxImage:k8s.gcr.io/pause:3.6 StatsCollectPeriod:10 SystemdCgroup:false EnableTLSStreaming:false X509KeyPairStreaming:{TLSCertFile: TLSKeyFile:} MaxContainerLogLineSize:16384 DisableCgroup:false DisableApparmor:false RestrictOOMScoreAdj:false MaxConcurrentDownloads:3 DisableProcMount:false UnsetSeccompProfile: TolerateMissingHugetlbController:true DisableHugetlbController:true DeviceOwnershipFromSecurityContext:false IgnoreImageDefinedVolumes:false NetNSMountsUnderStateDir:false EnableUnprivilegedPorts:false EnableUnprivilegedICMP:false} ContainerdRootDir:/var/lib/containerd ContainerdEndpoint:/run/containerd/containerd.sock RootDir:/var/lib/containerd/io.containerd.grpc.v1.cri StateDir:/run/containerd/io.containerd.grpc.v1.cri}""",1
b5e799c5,"time=""2023-11-09T13:11:39.698413611Z"" level=error msg=""failed to load cni during init, please check CRI plugin status before setting up network for pods"" error=""cni config load failed: no network config found in /etc/cni/net.d: cni plugin not initialized: failed to load cni config""",1
43c28e5a,error finding row index in _ifXTable_container_row_restore,5
2e7364a6,Errors from xkbcomp are not fatal to the X server,2
e947cb0d,systemd-journald.service: Attaching egress BPF program to cgroup /sys/fs/cgroup/unified/system.slice/systemd-journald.service failed: Invalid argument,1
4f49c93f,"Start signalR Config: {""HandshakeTimeout"":""00:00:15"",""KeepAliveInterval"":""00:00:10"",""ClientTimeoutInterval"":""00:02:00"",""SupportedProtocols"":[""json"",""json""],""MaximumReceiveMessageSize"":32768,""EnableDetailedErrors"":true,""StreamBufferCapacity"":10,""MaximumParallelInvocationsPerClient"":1}",5
2b76765b,ERROR: Cannot create file <*> Permission denied,6
aa61e506,error: kex_exchange_identification: Connection closed by remote host,630
b3901e8f,CPDefErrorLoggerMulti::Write(): No Loggers registered!,10
e2f341f3,"ERROR: Cannot create file ""PU_1100/PU_1100_mic_serial.log"": No such file or directory",1
1865a981,"libGL error: MESA-LOADER: failed to open radeonsi: /usr/lib/dri/radeonsi_dri.so: cannot open shared object file: No such file or directory (search paths /usr/lib/dri, suffix _dri)",1
d26df6b5,libGL error: failed to load driver: radeonsi,2
18c97926,libGL error: failed to open /dev/dri/card0: No such file or directory,1
ddff9a5b,[88:88:1109/131145.216235:ERROR:gl_surface_egl.cc(837)] EGL Driver message (Error) eglInitialize: DRI2: failed to find EGLDevice,1
c4864afc,"[88:88:1109/131145.282933:ERROR:gpu_init.cc(481)] Passthrough is not supported, GL is egl, ANGLE is",1
88f5b8e5,[114:9:1109/131145.512255:ERROR:command_buffer_proxy_impl.cc(125)] ContextResult::kTransientFailure: Failed to send GpuControl.CreateCommandBuffer.,1
9606b350,CMX_COMMON_IOCTL_RESET_CST_ERROR_MONITOR call,7
f11d4f18,FileNotFoundError: [Errno 2] No such file or directory: 'FrontUnit_1500/setup.cmxsetup',1
3cdb6887,"[warning ] Failed GET http://localhost:7777/notebooks/tree?token=instrument (elapsed=0.00508): ConnectionError(MaxRetryError(""HTTPConnectionPool(host='localhost', port=7777): Max retries exceeded with url: /notebooks/tree?token=instrument (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7d49519630>: Failed to establish a new connection: [Errno 111] Connection refused'))"")) context=_setup | get_jupyter_notebook_status | is_running pathname=/usr/lib/callysto/venv/lib/python3.10/site-packages/rs_callysto/notebook_server_controller.py lineno=659",1
f31c60aa,<*> ../../pcsc-lite-1.9.0/src/winscard.c:286:SCardConnect() Error Reader Exclusive,60
4f52a2a8,RsDeviceStarter[3219]: WARNING: in CRSProductManager.cpp at HandleStartupWarning():2970: [SWP_OPM_Result] Startup Warnings occurred! Please check log file '/home/instrument/fw/log/2023-11-09T13-11-48Z/StartupErrors.txt',1
27a0c988,WARNING: in CRSProductManager.cpp at HandleStartupWarning():2970: [SWP_OPM_Result] Startup Warnings occurred! Please check log file '/home/instrument/fw/log/2023-11-09T13-11-48Z/StartupErrors.txt',1
d9ff4af7,RsDeviceStarter[3219]: WARNING: in CRSScpiUnitSuffix.cpp at SetOwnUnitCatId():256: [unknown thread name] pIBase != NULL Unit-Linkage is invalid,11
9cdb67e3,WARNING: in CRSScpiUnitSuffix.cpp at SetOwnUnitCatId():256: [unknown thread name] pIBase != NULL Unit-Linkage is invalid,11
507c2f4c,ERROR @/home/integration_mmi/jenkins/workspace/mrt.base.fsw_release_5.0.60-ci/1cm_fsw_base_wm/fwa/rs.cmwbase.fsw/fswselfalignment/task_service.cpp:309 Expected files do not exist!;/home/instrument/fw/system/CorrdataWS/RFU300/RFUNIT_1222.1020.02_100131/CDV/SA/V6/CUR/tx_meta.csv,1
7b5763f5,RegisterSecondaryAlfManager name:DAU address:<*> status:18 error:Invalid Setup error adding slave <*>,1
c9c7eab9,Socket: connect operation failed with error 110,2
7fe303f7,ERROR: Caught exception while processing cell add/update '//mrt/0/testenvironment/0/signaling/0/sri/0/asservice/0/nrservice/0/cell/0': Cell //mrt/0/testenvironment/0/signaling/0/sri/0/asservice/0/nrservice/0/cell/0 is missing rach_config_common,1
cbca8e61,ERROR: Recall failed: The given file does not exist: /home/instrument/fw/system/Restauration/PersistenceClients/csp_settings.dat,1
03f783fc,<*> [DeviceApiSubscriber ] ERROR: ----------------------------------------------------------------------,32
bf0492df,"<*> [DeviceApiSubscriber ] ERROR: MRDC Band Combination error: Could not parse the ""SUL"" in <*>",32
8515778b,<*> [DeviceApiSubscriber ] ERROR: MRDC Band Combination message:,32
16dd5682,2023-11-09T13:15:26.011816Z [EngineCommander ] ERROR: Unsupported Engine state Start,1
62037388,"PIAS: <*> << 0,""No error""\n",2
edd3dfa6,PIAS: Socket[3]: Error: Data channel closed with error 104,1
f8379ea6,"PIAS: <*> SCPI-Error: -113,""Undefined <*>",21
3b6c7910,PIAS: <*> Error: invalid command <*>,21
38dda80a,Invalid user <*> from <*> port <*>,7
31cd7a89,Connection closed by invalid user <*> <*> port <*> [preauth],7
21bb6d0e,Failed password for invalid user <*> from <*> port <*> ssh2,2
2702de18,Failed none for invalid user <*> from <*> port <*> ssh2,3
099a7b10,PIAS: Socket[7] >> DIAG:PIAS:ID?\PIAS: Error: accept() failed with error 22,1
823a9263,ERROR: Client(500: tcp_v6://[fe80::02e0:33ff:fe0b:b5bf%eth_internal]:21840): Connection broken (),1
b79b4e77,"RsDeviceStarter[3219]: WARNING: in CRsTimeLinux.cpp at GetNTPEnabled():443: [SWP_Timer] DBus error: org.freedesktop.systemd1.ShuttingDown (Refusing activation, D-Bus is shutting down.)",4
47d4bf45,"WARNING: in CRsTimeLinux.cpp at GetNTPEnabled():443: [SWP_Timer] DBus error: org.freedesktop.systemd1.ShuttingDown (Refusing activation, D-Bus is shutting down.)",4
29d441bc,Task.Delay error! TaskCanceledException: System.Threading.Tasks.TaskCanceledException: A task was canceled.,2
5e50a317,"<*> level=warning msg=""Failed to delete conntrack state for <*> invalid argument""",3
66a25f54,RsDeviceStarter[3219]: WARNING: in cdevicefootprintprovider.cpp at getNameSpaceVersions():45: [unknown thread name] RsAlf::AlfManagerConfigurationClient::getNamespaceVersions returnd error code: 2 - Connection to local AlfManager could not be initialized.,1
6d305684,WARNING: in cdevicefootprintprovider.cpp at getNameSpaceVersions():45: [unknown thread name] RsAlf::AlfManagerConfigurationClient::getNamespaceVersions returnd error code: 2 - Connection to local AlfManager could not be initialized.,1
e579c8c0,[28]last error:,1
7a929bcc,<ErrorCode>No error. Some restrictions apply.</ErrorCode>,2
914f879e,</Error>,2
b13f757f,"<Error ErrorLevel=""RepairedError"" Timestamp=""2023-11-10T05:49:09.063Z"">",1
3f59712f,</Errors>,1
036ae2b6,</CompassErrorLog>,1
bcd65e6e,Type = ERROR_REPAIR,1
b0af403d,"time=""2023-11-10T05:49:11.963466786Z"" level=info msg=""stopping event stream following graceful shutdown"" error=""<nil>"" module=libcontainerd namespace=moby",1
